README.md

```diff
@@ -1,311 +1,14 @@
 # Conseil
-Query API for the Tezos blockchain
 
 [![Build Status](https://travis-ci.org/Cryptonomic/Conseil.svg?branch=master)](https://travis-ci.org/Cryptonomic/Conseil)
 [![Coverage Status](https://coveralls.io/repos/github/Cryptonomic/Conseil/badge.svg?branch=master)](https://coveralls.io/github/Cryptonomic/Conseil?branch=master)
 
-## Running Conseil
+Conseil is an indexer and API for blockchains, currently focused on the [Tezos](https://tezos.com/), which allows you to seamlessly run complex queries against blockchain data. It is written in the functional style in the [Scala](https://scala-lang.org/) programming language. This project forms an essential part of the [Cryptonomic stack](https://cryptonomic.tech/developers.html).
 
+Conseil consists of two processes:
+- Lorre, which indexes the blockchain by downloading data from blockchain nodes and storing it in databases.
```

Comment:
`and stores it in database.` - I think that would be more accurate, Lorre uses one DB :)